go/cmd/topo2topo/topo2topo.go

```diff
@@ -60,70 +72,66 @@ func main() {
 	grpccommon.RegisterFlags(fs)
 	log.RegisterFlags(fs)
 	logutil.RegisterFlags(fs)
-	_flag.Parse(fs)
-	args := _flag.Args()
-	if len(args) != 0 {
-		flag.Usage()
-		log.Exitf("topo2topo doesn't take any parameter.")
-	}
 
-	fromTS, err := topo.OpenServer(*fromImplementation, *fromServerAddress, *fromRoot)
+	servenv.ParseFlags("topo2topo")
+
+	fromTS, err := topo.OpenServer(fromImplementation, fromServerAddress, fromRoot)
 	if err != nil {
-		log.Exitf("Cannot open 'from' topo %v: %v", *fromImplementation, err)
+		log.Exitf("Cannot open 'from' topo %v: %v", fromImplementation, err)
 	}
-	toTS, err := topo.OpenServer(*toImplementation, *toServerAddress, *toRoot)
+	toTS, err := topo.OpenServer(toImplementation, toServerAddress, toRoot)
 	if err != nil {
-		log.Exitf("Cannot open 'to' topo %v: %v", *toImplementation, err)
+		log.Exitf("Cannot open 'to' topo %v: %v", toImplementation, err)
 	}
 
 	ctx := context.Background()
 
-	if *compare {
-		compareTopos(ctx, fromTS, toTS)
+	if compare {
+		compareTopos(ctx, fromTS, toTS, doKeyspaces, doShards, doShardReplications, doTablets)
 		return
 	}
-	copyTopos(ctx, fromTS, toTS)
+	copyTopos(ctx, fromTS, toTS, doKeyspaces, doShards, doShardReplications, doTablets, doRoutingRules)
 }
 
-func copyTopos(ctx context.Context, fromTS, toTS *topo.Server) {
-	if *doKeyspaces {
+func copyTopos(ctx context.Context, fromTS, toTS *topo.Server, doKeyspaces bool, doShards bool, doShardReplications bool, doTablets bool, doRoutingRules bool) {
```

Comment:
why did this function signature change?

Reply:
nitially I declared these variables locally in main, therefore I passed on these variables to the helper functions. Now that I moved it outside main I have reverted these changes.